src/lib/Menu/UserSetting/UserSettingUpdateRightSidebarBuilder.php

```diff
@@ -90,7 +92,8 @@ public function createStructure(array $options): ItemInterface
             self::ITEM__CANCEL => $this->createMenuItem(
                 self::ITEM__CANCEL,
                 [
-                    'route' => 'ibexa.user_settings.list',
+                    'route' => $route,
```

Comment:
This route is passed to `UrlGeneratorInterface` next, when rendering the link. Invalid route (or a missing route parameter) might lead to page that this menu is on to fail to render.

We should ensure that this route is actually valid, before passing it to menu instance.

Reply:
Fixed in 75d271c9.